lib/command-palette-view.js

```diff
@@ -69,6 +70,14 @@ export default class CommandPaletteView {
       },
       didConfirmSelection: (keyBinding) => {
         this.hide()
+        const elementName = keyBinding.name
```

Comment:
This could use a better name. `elementName` makes it seem as though it's referring to the DOM.